internal/directory/google/google.go

```diff
@@ -94,7 +94,7 @@ func (p *Provider) UserGroups(ctx context.Context) ([]*directory.User, error) {
 		Customer("my_customer").
 		Pages(ctx, func(res *admin.Groups) error {
 			for _, g := range res.Groups {
-				groups = append(groups, g.Id)
+				groups = append(groups, g.Id, g.Email)
```

Comment:
I don't think this change is enough and works. In the for loop several lines below, we do a map from userid -> group id, by calling `apiClient.Member.List(group)`, using group id.

I think you need another map from groupID -> groupName, then append the groupName to `userIDToGroups` whenever you append corresponding groupID.

Reply:
It did work when I tested.  It seems List() accepts a few different formats - https://developers.google.com/admin-sdk/directory/v1/reference/members/list.

Should probably do something similar to your suggestion regardless, though.  We're now looking up each group's members twice.  Not ideal if you had, say 5000 groups 😄

Reply:
Related question, should we adjust "max results" to something like 200? The docs are very unclear what the default return value is.

Reply:
The default is 200:

```
Groups and group members – A default and maximum of 200 entries per page.
```

https://developers.google.com/admin-sdk/directory/v1/limits